README.md

```diff
@@ -20,6 +20,18 @@ This app, as well as the ready-to-distribute binaries (see [Deploy](#deploy)), i
 1. Install [homebrew](http://brew.sh/)
 2. `brew update`
 3. `brew install wine`
+4. Add a Windows build to your settings.
+
+```json
+{
+  "electron": {
+    "builds": [
+      {"platform": "win32",
```

Comment:
When the user starts to specify `builds`, do they have to include the Mac platform too?